integration-tests/com.liferay.ide.integration.testing-repository/category.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<site>
+	<bundle id="com.liferay.ide.core.base" version="0.0.0" />
```

Comment:
These bundles should have `test` in their names

`com.liferay.ide.test.core.base`
`com.liferay.ide.test.project.core.base`